src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,46 @@
+.bp-ColorPickerPalette {
+    position: absolute;
+    left: -55px;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 35px;
+    margin-bottom: 125px;
+    background: inherit;
+    border-radius: 5px;
+
+    &::after {
```

Comment:
Is this an arrow? Do any of our other popups have arrows? If not, should we omit this for now?

Reply:
Yep this is an arrow. I checked in with design and design confirmed that even though this breaks consistency, for this specific component we would like to keep the arrow there.